admin/services/data-access/pupil-status.data.service.js

```diff
@@ -0,0 +1,101 @@
+'use strict'
+
+const { TYPES } = require('./sql.service')
+const R = require('ramda')
+const sqlService = require('./sql.service')
+const pupilStatusDataService = {}
+
+pupilStatusDataService.sqlFindPupilsFullStatus = async function sqlFindPupilsAndAttendanceReasons (schoolId) {
```

Comment:
how comes the property and function name are different?

Reply:
good catch, fixed.